src/games/tic_tac_toe.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+exports.setUp = function setUp(players) {
```

Comment:
Oh and I meant that as an assert that the length of players should always be 2 here.